src/main/scala/com/raphtory/core/components/graphbuilder/GraphMessages.scala

```diff
@@ -2,21 +2,72 @@ package com.raphtory.core.components.graphbuilder
 
 import com.raphtory.serialisers.avro
 import org.apache.pulsar.client.api.Schema
-
-/** @DoNotDocument */
-sealed trait Property {
-  def key: String
-  def value: Any
+import Properties._
+
+/**
+  * # Properties
+  *
+  * Vertex/Edge properties
```

Comment:
please add a short line/para explaining what properties are for,

Reply:
Done